tests/core_tests.py

```diff
@@ -256,6 +256,23 @@ def test_shortner(self):
         resp = self.client.post('/r/shortner/', data=data)
         assert '/r/' in resp.data.decode('utf-8')
 
+    def test_kv(self):
+        self.logout()
+        self.login(username='admin')
+        data = {
+            'data': 'this is a test',
+        }
+        resp = self.client.post('/kv/store/', data=dict(data=json.dumps(data)))
+        kv = db.session.query(models.KeyValue).first()
+        kv_value = kv.value
```

Comment:
I would prefer you to  get id from response in this test, 
often tests are used as documentation for the functions, to display how they should be used.